tests/dash_core_components_page.py

```diff
@@ -0,0 +1,64 @@
+import logging
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+logger = logging.getLogger(__name__)
+
+
+class DashCoreComponentsMixin(object):
+    def select_calendar_date(
```

Comment:
@alexcjohnson  the selector is now a property, would this be generic enough to be ported?

Reply:
Personally I'd prefer to still leave it in this repo. If we find another use for it outside DCC, *and* it still works in the form here, it should be easy to move over at that point; but it feels quite likely to me that it's still tightly coupled to this particular datepicker implementation - the start/end stuff, the `__outside`...

Regardless, looks great this way :)